apps/frontend/src/api/__tests__/get-list.test.tsx

```diff
@@ -16,17 +16,35 @@ jest.mock('../../config');
 
 describe('createListApiUrl', () => {
   it('handles requests without parameters ', async () => {
-    expect(createListApiUrl('test').toString()).toMatch(/\/test$/);
+    expect(
+      createListApiUrl('test', {
+        pageSize: null,
+        currentPage: null,
+      }).toString(),
+    ).toMatch(/\/test$/);
+  });
+
+  it('sets default page and size', async () => {
+    const url = createListApiUrl('test');
+    expect(url.searchParams.get('take')).toEqual('10');
+    expect(url.searchParams.get('skip')).toEqual('0');
   });
+
+  it('calculates take and skip from params', async () => {
+    const url = createListApiUrl('test', { currentPage: 2, pageSize: 10 });
+    expect(url.searchParams.get('take')).toEqual('10');
+    expect(url.searchParams.get('skip')).toEqual('20');
+  });
+
   it('handles requests with a search query ', async () => {
     expect(
       createListApiUrl('test', { searchQuery: 'test123' }).toString(),
-    ).toContain('test?search=test123');
+    ).toContain('search=test123');
```

Comment:
I think one of the comments was not fully addressed here, can we check this properly using the URL's `searchParams`